### ISSUE TITLE
A few issues

### ISSUE THREAD

**Issue by TotallyInformation** [node-red/node-red-admin#6]
Hi, just discovering a few issues with node-red-admin.

I'm using Node-RED v0.19.4, node-red-admin v0.1.3. NodeJS v8.12.

1. `node-red-admin search uibuilder` (or any other search term of course) always returns a 404 and nothing else.

    ```bash
    pi@pi3:~/nrlive $ node-red-admin search telegram
    
    404:
    ```

2. On my Pi3, which has Node-RED using a self-signed certificate for https, if I do any command that needs to look up a Node-RED API end-point, I always get the following error:

    ```bash
    pi@pi3:~/nrlive $ node-red-admin list
    
    Error: self signed certificate in certificate chain
    ```

**Comment by biancode** [comment]
1. needs a new api call `https://registry.npmjs.org/-/v1/search?text=keywords:node-red+telegram`

**Comment by biancode** [comment]
https://registry.npmjs.org/-/v1/search?text=keywords:node-red+opcua

**Comment by biancode** [comment]
It could get closed state with v0.1.8
